examples/tooling/README.md

```diff
@@ -0,0 +1,38 @@
+MariaDB Example
+===============
+
+This example exists primarily to test the following documentation:
+
+* [MariaDB Service](https://docs.devwithlando.io/tutorials/mariadb.html)
+
+Start up tests
+--------------
+
+Run the following commands to get up and running with this example.
+
+```bash
+# Should start up successfully
+lando poweroff
+lando start
+```
+
+Verification commands
+---------------------
+
+Run the following commands to validate things are rolling as they should.
+
+```bash
+# Should not see deprecation warning on db-export
+lando db-export-test | ( ! grep "Deprecated" )
```

Comment:
While this is valid, I'd probably use, `grep --invert-match "Deprecated"` (`grep -v`). I like that vs inverting.

Reply:
@rubenvarela Thanks for taking a look. I had tried that but I was missing `grep -vz` to treat the entire input as a single line. Fixed in dc4eff7.